tests/integrations/utils.py

```diff
@@ -1,265 +1,46 @@
 import os
-import random
-import re
 import shutil
 import subprocess
-import tempfile
-import time
-from pathlib import Path
-from typing import Iterable, Optional
 
-import pytest
 import requests
-import yaml
 from requests.adapters import HTTPAdapter, Retry
 from scaleway import Client
-from scaleway.account.v2 import AccountV2API
-from scaleway.function.v1beta1 import FunctionV1Beta1API
-from scaleway.registry.v1 import RegistryV1API
-from scaleway_core.api import ScalewayException
-from typing_extensions import Self
 
-from scw_serverless.dependencies_manager import DependenciesManager
+from tests import constants
 
-from ..utils import COLD_START_TIMEOUT
-
-DEFAULT_REGION = "pl-waw"
 CLI_COMMAND = "scw-serverless"
 
 
-class ServerlessTestProject:
-    """
-    ServerlessTestProject encapsulates the logic of
-    setting up a clean integration test environment.
```

Comment:
I'm sorry for changing this again but I'm way happier with this new design. Last time I kinda gave up and decided to put everything in a single file and hide away this code as much as possible.

Now, I have separated the create project fixture from the CLI wrappers. To me, it seems a lot cleaner and makes it easier to manage.